src/app/campaign/campaign.component.ts

```diff
@@ -81,29 +79,20 @@ export class CampaignComponent implements OnInit, OnDestroy {
         })
       )
       .subscribe(({ params, advertisers, accounts }) => {
-        this.campaignStoreService.load(params.get('id'));
         if (params.get('id')) {
           this.store.dispatch(new actions.CampaignLoad({ id: +params.get('id') }));
         } else {
           this.store.dispatch(new actions.CampaignNew());
         }
       });
-    this.campaignFlights$ = this.campaignStoreService.flights$.pipe(
-      map(flightStates => {
-        return Object.keys(flightStates).map(
-          (id): LocalFlightState => {
-            return {
-              id,
-              name: flightStates[id].localFlight.name || '(Flight)',
-              softDeleted: !!flightStates[id].softDeleted,
-              changed: flightStates[id].changed,
-              valid: flightStates[id].valid,
-              statusOk: !flightStates[id].localFlight.status || flightStates[id].localFlight.status === 'ok'
-            };
-          }
-        );
-      })
-    );
```

Comment:
The different states here are (see template above):
Loaded: Campaign and Flights have been new'ed or loaded via the api (show content or check error)
Loading: API is currently loading (show spinner)
Saving: Campaign and/or flights are saving. (show spinner on Save button)
Error: Campaign 404 or Flight not found on campaign (if loaded and error, show error)